### PR TITLE
Fix unit error in RobotR3.OneAxis

### PR DESCRIPTION
**Pull request by henrikt-ma**

### CODE REVIEW

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/OneAxis.mo [hunk 1]
```diff
@@ -4,6 +4,7 @@ model OneAxis
 
   extends Modelica.Icons.Example;
   parameter SI.Mass mLoad(min=0)=15 "Mass of load";
+  parameter SI.Length rg = sqrt(1.3) "Radius of gyration of load";
```
Comment (tobolar) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/OneAxis.mo:7]: `SI.Radius` would fit better here since rg>=0 m.

**Review by tobolar — CHANGES_REQUESTED** [review]

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/OneAxis.mo (hunk 1 above)
Comment (henrikt-ma) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/OneAxis.mo]: OK.  Done.  Resolving.

**Review by hubertus65 — APPROVED** [review]

**Comment by HansOlsson** [comment]
Hmm... Two related questions:

- Is "radius of gyration" clear enough as description?
- Is there some corresponding "area" that we can use to avoid taking the square-root and then squaring it?

**Comment by tobolar** [comment]
> Is "radius of gyration" clear enough as description?

I think "radius of gyration" is fine and known in the corrsponding theory (even if I personally forgot it years ago :-O ). An internet search also responds sufficient related stuff.

> Is there some corresponding "area" that we can use to avoid taking the square-root and then squaring it?

Not sure about this. A solution could be to take a value of radius of gyration close to sqrt(1.3), such as e.g. 1.14 m. This would lead to changes of load's inertia `load.J` of 0.03 %. Probably this would even be within the tolerance or regression checks.

**Comment by henrikt-ma** [comment]
> > Is there some corresponding "area" that we can use to avoid taking the square-root and then squaring it?
> 
> Not sure about this. A solution could be to take a value of radius of gyration close to sqrt(1.3), such as e.g. 1.14 m. This would lead to changes of load's inertia `load.J` of 0.03 %. Probably this would even be within the tolerance or regression checks.

Sounds good to me.  @tobolar, @HansOlsson, @hubertus65, @MartinOtter, please react to this comment with thumb up or thumb down as an indication of whether to go with 1.14.

**Review by hubertus65 — APPROVED** [review]

**Comment by Harisankar-Allimangalath** [comment]
@MartinOtter can you please reply to @henrikt-ma  and do the pending review , Thankyou

**Comment by henrikt-ma** [comment]
I acted on the current reactions to https://github.com/modelica/ModelicaStandardLibrary/pull/4108#issuecomment-1510939157, which where 3/3 thumbs up, and with this I hope that this is now ready to be merged.

**Comment by tobolar** [comment]
> ...  I hope that this is now ready to be merged.

I fully agree. We have two positive reviews and could merge.

**Pull request merged**